docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -50,6 +50,14 @@ To change the width of a column, use the `width` property available in `GridColD
 
 {{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline"}}
 
+### Minimum width
+
+By default, the columns have a minimum width of 50 pixels.
+This is an arbitrary, easy to remember value.
```

Comment:
https://english.stackexchange.com/questions/293323/easy-to-use-versus-easy-to-use/293336

```suggestion
This is an arbitrary, easy-to-remember value.
```